File: app/src/main/java/com/example/devicemanager/activity/LoginActivity.java
package com.example.devicemanager.activity;

import android.os.Bundle;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.example.devicemanager.R;
import com.example.devicemanager.fragment.LoginFragment;

public class LoginActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

        if(savedInstanceState == null){
            getSupportFragmentManager().beginTransaction()
                    .replace(R.id.contentContainer, LoginFragment.newInstance())
                    .commit();
        }
    }

    @Override
    public void onBackPressed() {
        super.onBackPressed();
        finish();
    }
}

File: app/src/main/java/com/example/devicemanager/activity/LoginActivity.kt
package com.example.devicemanager.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import com.example.devicemanager.R
import com.example.devicemanager.fragment.LoginFragment

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction()
                    .replace(R.id.contentContainer, LoginFragment.newInstance())
                    .commit()
        }
    }

    override fun onBackPressed() {
        super.onBackPressed()
        finish()
    }
}
